doc/releasenotes/16_0_0_summary.md

```diff
@@ -144,6 +149,28 @@ The `RestoreFromBackup  --restore_to_pos` ends with:
 - the restored server in intentionally broken replication setup
 - tablet type is `DRAINED`
 
+### Error codes
+
+In this release, we are introducing a new way to handle errors from Vitess through the query interface. 
+Errors will now have an error code for each error, which will make it easy to search for more information on the issue.
+Example:
+The error 
+
+```
+aggregate functions take a single argument 'count(user_id, name)'
+```
+
+will now instead be:
+
+```
+VT03001: aggregate functions take a single argument 'count(user_id, name)'
+```
+
+The error code `VT03001` can then be used to search or ask for help and report problems.
+
+If you have code searching for error strings from Vitess, this is a breaking change. 
+Many error strings have been tweaked.
```

Comment:
This seems to be repeated? We are already saying that in the "error strings" section.

Reply:
resolved in [f2351bb](https://github.com/vitessio/vitess/pull/10738/commits/f2351bba65f347a8e5ce72ff87757ccaf20115af), merged the two sections into a single one under `Error Strings`